README.md

```diff
@@ -4,10 +4,84 @@ This repository contains a Javascript implementation of [esptool](https://github
 
 `esptool-js` is based on [Web Serial API](https://wicg.github.io/serial/) and works in Google Chrome and Microsoft Edge, [version 89 or later](https://developer.mozilla.org/en-US/docs/Web/API/Serial#browser_compatibility).
 
+
+## Usage
+
+CDN
+
+``
```

Comment:
CDN link that you can use is `https://unpkg.com/esptool-js/lib/index.js?module`

Note that you need to hardcode the path to the import file. Otherwise it will 304 to the right file, but the JavaScript that loaded it won't know, so all relative import from `index.js` will fail.